client/internal/http/version.go

```diff
@@ -0,0 +1,3 @@
+package http
+
+var Version = "localdevelopment"
```

Comment:
I would add a comment: `this will be overwritten during compilation time, see: provider/.goreleaser.yml`

Reply:
Added :)